internal/js/modules/k6/browser/tests/test_browser.go

```diff
@@ -271,6 +271,18 @@ func (tb *testBrowser) withIFrameURL(iframeURL string, iframeID string) {
 	})
 }
 
+// newPageWithIFrameSrcdoc creates a new page with an inline iframe using srcdoc.
```

Comment:
Thanks for abstracting out the common logic. Could you find elsewhere in tests that this could be used? I don't see why we would need to abstract logic like this in a separate function otherwise.